### PR TITLE
Enabled sorting workflow definitions and instance search result by clicking column header

### PR DESCRIPTION
**Pull request by eputtone**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,8 @@
 **Highlights**
 
 **Details**
+- `nflow-explorer`
```
Comment (efonsell) [inline CHANGELOG.md:6]: could be mentioned in highlights :)

File: nflow-explorer/src/app/search/searchResult.html [hunk 2]
```diff
@@ -3,23 +3,23 @@ <h2>Workflows</h2>
...
-    <th>State text</th>
-    <th>Status</th>
-    <th>Business key</th>
-    <th>External id</th>
-    <th>Retries</th>
-    <th>Created</th>
-    <th>Started</th>
-    <th>Modified</th>
-    <th>Next activation</th>
+    <th st-sort="id">Id</th>
```
Comment (efonsell) [inline nflow-explorer/src/app/search/searchResult.html:8]: Would be nice if mouse cursor would indicate clickable header somehow?

File: CHANGELOG.md (hunk 1 above)
Comment (eputtone) [reply CHANGELOG.md]: Now it's in the highlights

File: nflow-explorer/src/app/search/searchResult.html (hunk 2 above)
Comment (eputtone) [reply nflow-explorer/src/app/search/searchResult.html]: Table header looks clickable now. Also added styles for displaying sort order direction (didn't notice that before).

**Review by efonsell — APPROVED** [review]

**Pull request merged**